changelog.md

```diff
@@ -15,6 +15,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!-- ## [Unreleased] -->
 
 ## Released
+## [2.3.7] - 2023-07-19
```

Comment:
You can keep the changelog as it is right now, I'll add the necessary changes after the merge :)

Reply:
Should I revert the changes? These were after I pulled from upstream and resolved the merge conflicts, I think if I change it back now, then there'll be problems when this pull request is auto-merged by github.